doc/requirements.rst

```diff
@@ -18,7 +18,7 @@ Python 3.7 or older.
 macOS
 ~~~~~~
 
-|PyInstaller| runs on macOS 10.13 (High Sierra) or newer.
+PyInstaller runs on macOS 10.13 (High Sierra) or newer.
```

Comment:
Let's make that 10.15 (Catalina) - earlier versions are EOL'd by now, anyway. We can add a note that 10.13 could still work, but likely requires installation from sdist and a bootloader built with correspondingly old toolchain because the `codesign` utility on 10.13 seems to do weird stuff with signatures generated by newer toolchain (and seems to be buggy in general)...

Reply:
I don't think the note is necessary, swapped the version though.